tox.ini

```diff
@@ -1,19 +1,10 @@
 [tox]
-envlist = py36,py37,py38,py39,linting
+envlist = py38,py39,py310,py311,py312
 
 [testenv]
 deps =
     pytest-xdist
 commands = pytest {posargs:tests}
 
-[testenv:linting]
-skip_install = True
-basepython = python3.7
-deps = pre-commit>=1.11.0
-commands = pre-commit run --all-files --show-diff-on-failure
-
-[flake8]
-max-line-length = 100
```

Comment:
Just to know, is this not necessary anymore? Linting is being called in another way?

Reply:
I decided to drop it and let "black" take care of that for us, and in case some comment goes over that border a bit I do not find a big deal either.